.circleci/config.yml

```diff
@@ -127,7 +127,7 @@ commands:
 
                 command: |
                   mv .riot .ddriot
-                  echo -p2.7,-p3.5,-p3.6,-p3.7,-p3.8,-p3.9 | tr ',' '\n' | circleci tests split | xargs ./scripts/ddtest riot -v run --exitfirst --pass-env -s '<< parameters.pattern >>'
+                  riot run -s avail_pythons | head -n1 | tr ',' '\n' | circleci tests split | xargs -I PY ./scripts/ddtest riot -v run --python=PY --exitfirst --pass-env -s '<< parameters.pattern >>'
```

Comment:
note that the `head -n1` is necessary as riot includes the standard after test result summary (and I don't think we have an option to skip it).